package.json

```diff
@@ -11,7 +11,7 @@
     "format:check": "prettier --check ."
   },
   "dependencies": {
-    "@livekit/components-react": "^2.9.15",
+    "@livekit/components-react": "0.0.0-agents-sdk-20251113151554",
```

Comment:
nitpick: This is more of an action item for me, but IMO the agents sdk should be published to a mainline release before this is merged. I don't think there is anything that is blocking this so I'll try to get this done in the next 12-24 hours and follow up here once I have so you can update the version here.

Reply:
Update: `@livekit/components-react@2.9.16` was just released which contains the new agents sdk features. If you can update to use this new version instead of the `agents-sdk` tag then I think I'm good with this getting merged 👍